Services/DataBaseService/Utils/MigrationScripts/24.03.2020/1_CreateUsersTable.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [Users] (
```

Comment:
Will we create another table? Right now it is UserCredentials table, I think

Reply:
You`re right. I fix this